scripts/cpu_info.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+get_percent()
+{
+	case $(uname -s) in
+		Linux)
+			percent=$(top -bn1 | grep "Cpu(s)" | sed "s/.*, *\([0-9.]*\)%* id.*/\1/" | awk '{print 100 - $1"%"}')
```

Comment:
So the names for these cases wouldn't be Linux and Mac anymore. Check out network.sh or battery.sh as an example.

Reply:
Oops, you're right.